Add note on target temp validation on ClimateEntity (#2299)
home-assistant/developers.home-assistant: docs/core/entity/climate.md

### Set temperature

:::note

> `ClimateEntity` has built-in validation that the target low temperature is lower than equal to target high temperature so integrations does not need to validate it in their own implementation.

Author: ```suggestion
`ClimateEntity` has built-in validation to ensure that the target low temperature is lower than or equal to the target high temperature. Therefore, integrations do not need to validate this in their own implementation.
```

> `ClimateEntity` has built-in validation that the target low temperature is lower than equal to target high temperature so integrations does not need to validate it in their own implementation.

Reviewer: What happened to the previous version and my suggestion? I suggest we bring my suggestion back. 🙂

Author: Yeah. My rebase erased that previous change by accident. I will change it again 👍

> `ClimateEntity` has built-in validation to ensure that the target low temperature is lower than or equal to the target high temperature. Therefore, integrations do not need to validate this in their own implementation.

Reviewer: ```suggestion
`ClimateEntity` has built-in validation to ensure that the `target_temperature_low` argument is lower than or equal to the `target_temperature_high` argument. Therefore, integrations do not need to validate this in their own implementation.
```
